docs/index.rst

```diff
@@ -0,0 +1,57 @@
+.. rasterio documentation master file, created by
+   sphinx-quickstart on Thu Mar 17 07:05:00 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Rasterio Documentation
+======================
+
+.. warning::
+    This is not the official documentation. Not yet.
+    This is a draft and everything here is subject to change. 
+    
+    For now, please refer to https://github.com/mapbox/rasterio
+    for documentation.
+
+Rasterio is for Python programmers and command line users
+who want to read, write and manipulate geospatial raster datasets.
+
+Rasterio employs GDAL_ under the hood for file I/O and raster formatting.
+Its functions typically accept and return Numpy ndarrays.
+Rasterio is designed to make working with geospatial raster data more productive and more fun.
+
+You can download from pypi and todo quick link to install
```

Comment:
@perrygeo is this TODO intentional here, or can it be linked to install page now?